invisible_cities/cities/components.py

```diff
@@ -152,6 +152,17 @@ def print_every_loop(target):
     return print_every_loop
 
 
+def collect():
+    """
+    Collect into a list an argument (selector string) passed through a pipe.
+    The list can be retrieved from a future.
```

Comment:
I wouldn't mention the selector string, as it's completely orthogonal: `collect` collects whatever comes down the pipe, which might or might not have been selected with a selector string.

Saying that it can be retrieved from a future also seems a bit odd, because this is absolutely standard behaviour for any sink that produces a value. So, either don't mention the future at all, or describe fully how it returns a a future and a sink pair.

Also, https://www.python.org/dev/peps/pep-0257/#id17

OTOH, I wouldn't worry too much about all this, as it will be documented properly in the stand-alone dataflow package. So you might as well leave it as it is, for the purposes of this PR.